jungfrau_gui/decoder.py

```diff
@@ -0,0 +1,74 @@
+import zmq
+import cbor2
+from .dectris.compression import decompress
```

Comment:
Should this not be
```python
from dectris.compression import decompress
```
without the leading dot?

Reply:
This works as I built the compression (.so) package from scratch in the "broken" (dev) env and put in the dectris folder... Will fix the env and make the changes as you suggest